tensorflow/java/src/main/java/org/tensorflow/Tensor.java

```diff
@@ -348,21 +433,21 @@ public boolean booleanValue() {
    * @throws IllegalArgumentException if the tensor is a scalar or if {@code dst} is not compatible
    *     with the tensor (for example, mismatched data types or shapes).
    */
-  public <T> T copyTo(T dst) {
+  public <U> U copyTo(U dst) {
```

Comment:
The type parameter U here is not the type parameter of the `Tensor` type itself, but rather a parameter of the method `copyTo`. A different name is used to avoid confusing these two types.